python/pyspark/sql/readwriter.py

```diff
@@ -180,6 +180,9 @@ def json(self, path, schema=None, primitivesAsString=None, prefersDecimal=None,
         :param path: string represents path to the JSON dataset,
                      or RDD of Strings storing JSON objects.
         :param schema: an optional :class:`StructType` for the input schema.
+        :param samplingRatio: sets the ratio for sampling and reading the input data to infer
```

Comment:
Ah, I see. It does not affect the actual I/O but just drops some and then try to infer the schema. 
I will remove the change.

BTW, actually, I have found another one [`mergeSchema`](https://github.com/apache/spark/blob/431542765785304edb76a19885fbc5f9b8ae7d64/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala#L148-L152) option in Parquet data source, which I guess should be located in `ParquetOptions`. Can this be done here together maybe..?